CHANGELOG.md

```diff
@@ -8,13 +8,16 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ## [unreleased] - yyyy-mm-dd
```

Comment:
We've left it in this file in the past, and simply used the changelog entries for the specific release version in the github release changelog. But I think you make a good point, it would be cleaner to remove it on our main branch.

Reply:
i see you removed it, perfect!